src/ui/api/project.go

```diff
@@ -347,10 +347,10 @@ func (p *ProjectAPI) List() {
 					return
 				}
 				projects = append(projects, pros...)
-
 				mps, err := p.ProjectMgr.List(&models.ProjectQueryParam{
 					Member: &models.MemberQuery{
-						Name: p.SecurityCtx.GetUsername(),
+						Name:      p.SecurityCtx.GetUsername(),
+						GroupList: p.SecurityCtx.GetUserGroupList(),
```

Comment:
If we call SecurityCtx.GetMyProjects() directly, we can hide the getting user group list logic inside GetMyProjects and do not need the GetUserGroupList anymore?

Reply:
change to SecurityContext.GetMyProjects()